src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
```

Comment:
The variable name 'cost' is not clear enough. Rename it to 'rentCostPerDay' to indicate it's the rent cost per day. 📚